src/exceptions/HttpError.js

```diff
@@ -0,0 +1,17 @@
+import log from '../utils/log';
+
+export default class HttpError extends Error {
+  constructor(_code, _message) {
```

Comment:
Underscore dangles in constructor variables are not usual practice in JS. The only time they are used are for _private_ items, such as ```__xyz__``` from SpiderMonkey